src/chainstate/stacks/boot/mod.rs

```diff
@@ -3460,7 +3460,7 @@ pub mod test {
 
                 if cur_reward_cycle >= alice_reward_cycle {
                     // this will grow as more miner rewards are unlocked, so be wary
-                    if tenure_id >= (MINER_REWARD_MATURITY + MINER_REWARD_WINDOW + 1) as usize {
+                    if tenure_id >= (MINER_REWARD_MATURITY + 1) as usize {
```

Comment:
Same here

Reply:
`MINER_REWARD_MATURITY` is a `u64` because it is used with `u64` block-height values in non-test code.  If avoiding `as X` in non-test code requires that unit tests have to use `as X`, then that's a trade-off I'm willing to make.  Also, making a protocol constant a `usize` is risky, because `usize` does not have a guaranteed number of bits in its representation (whereas `u64` does).